README.md

```diff
@@ -14,21 +13,60 @@ Takeaway Challenge
 
  ```
 
-Instructions
--------
-
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
-
-Task
------
-
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+## Task
+I have been asked to write a Takeaway program that will include a menu with a list of dishes with their respective prices. People will be able to use the program to place their orders, via selecting the dishes they want and their respective quantities. The program will work out the total cost of their order and I will use Twilio in order to send text messages to let people know when their order will be ready.\
+\
+During testing, I will try to use stubs and doubles in order to isolate my tests.\
+\
+Edge cases considered:
+- Customers will not be able to order items not on the menu
+- Customers have the ability to cancel orders
+- Customers can not create a new order until the previous one has been placed or cancelled
+- Customer will get a error if they try to place order without having created one yet
+
+## Instructions
+
+Clone this repository to your desired location, run the command `gem install bundler` then run `bundle`.\
+\
+Run RSpec in the `takeaway_challenge` directory whilst in the terminal in order to run the unit tests.\
+\
+Run irb and load the file `takeaway.rb` inside the `lib` directory. Create a new takeaway within irb by using `Takeaway.new`. Use the command `show_menu` in order to see the menu, alternatively, if you use the command `take_order`, after inputting your 10 digit mobile phone number it will automatically show you the menu. Follow the on-screen instructions to place your order and it will show you what you have ordered when it asks to confirm your order. Input `yes` to confirm it, it will then show you the total cost.\
+\
+If you forget the cost of your order after confirming it, you can use the command `cost`. You can then use the command `place_order` in order to finalise it, which will send you a text message with the time it will be delivered by. If you wish to cancel your order instead, use the command `cancel_order`. Once you have placed or cancelled the order, you will be able to start on a new order. If you try to `take_order` having already confirmed a order, it will simply show you the cost again - if you wish to create a new order, you must do either `cancel_order` or `place_order`.\
+\
```

Comment:
Example of using the code in irb makes it clear to the user what they should be expecting